docs/patterns/existing-eks-observability-accelerators/existing-eks-awsnative-observability.md

```diff
@@ -102,7 +102,7 @@ prometheus-node-exporter   Active   13m
 - On a same view, select 'EKS Pods', which provides insights overview of all the pods as shown below
 ![Container Insights - EKS Pods](../images/awsnative-container-insights-2.png)
 
-- Please see [Single New EKS Cluster AWS Mixed Observability Accelerator](../single-new-eks-observability-accelerators/single-new-eks-mixed-observability.md#viewing-logs) for Logs.
+- Please see [Logging](../../logs.md) for information on viewing Logs.
```

Comment:
This change is missing in all other patterns. We want all patterns that logs to point to this page. like OSS, Mixed, AWS native in both single new and existing patterns

Reply:
Linked to main log page in each of the logging subsections for each pattern.